drivers/docker/handle.go

```diff
@@ -246,6 +246,8 @@ func (h *taskHandle) run() {
 	if ierr != nil {
 		h.logger.Error("failed to inspect container", "error", ierr)
 	} else if container.State.OOMKilled {
+		h.logger.Error("OOM Killed", "container_id", h.containerID, "container_image", h.containerImage, "docker_command", h.task.Command, "docker_command_args", h.task.Args)
```

Comment:
Hey @vxjohanforssell, sadly this won't even compile. `*drivers.TaskConfig` has no field or method `Command` or `Args`.

Reply:
I'm sorry, that was a waste of your time! 

I was editing through the web interface and I managed to look at the wrong struct when picking things to print. I've updated it now.